grpc-interface-support/src/cpp.py

```diff
@@ -54,7 +54,7 @@ def get_header_stub_code(self, include_path: str = ".") -> List[str]:
         grpc_header_path = os.path.join(
             self.__base_path,
             include_path,
-            f"{self.__proto_file.get_service_name().lower()}.grpc.pb.h",
+            f"{Path(self.__proto_file.file_path).stem}.grpc.pb.h",
```

Comment:
Tis does not match with `service_name` in line 74 if the file is called different. Could lead to not finding the correct files for conan package generation,

Reply:
this is true and intended, this one is the file-name which is always the same as the proto-input. the other one is the generated class which correspond to the service name defined inside of the proto.

See the tests exactly this is tested also. The seat service has equal filename and servicename whereas the door service has different names and both get generated successfully